Makefile

```diff
@@ -44,12 +45,31 @@ $(call add-bindata,assets,./bindata/...,bindata,assets,pkg/operator/assets/binda
 $(call build-image,cert-manager-operator,$(IMAGE_OPERATOR),./images/ci/Dockerfile,.)
 $(call build-image,cert-manager-operator-bundle,$(IMAGE_OPERATOR_BUNDLE),./bundle/bundle.Dockerfile,./bundle)
 
+# generate scorecard image targets
+.PHONY: build/custom-scorecard-tests
+build/custom-scorecard-tests:
+	go build -mod=vendor $(GO_GCFLAGS) $(GO_ASMFLAGS) -o $(BUILD_DIR)/$(@F) ./test
+
+.PHONY: build-e2e-image
+build-e2e-image:
+	docker build -f ./images/custom-scorecard-tests/Dockerfile ./ -t quay.io/ytripath/cert-manager-operator-e2e-tests:latest
+	docker push quay.io/ytripath/cert-manager-operator-e2e-tests:latest
+
+.PHONY: run-e2e-image
+run-e2e-image: test-e2e-wait-for-stable-state
+	oc apply -f test/rbac.yaml
+	operator-sdk scorecard ./bundle --namespace=cm-tests --wait-time=600s -o json
```

Comment:
Namespace and timeouts should be parametrized.

Reply:
This target is just used for local testing, also, we are setting up rbac in the `test/rbac.yaml` file for the tests to run, which includes the namespace, parameterizing the namescape will make this complex

Reply:
substituted with oc commands